src/wallet/walletdb.cpp

```diff
@@ -104,6 +104,26 @@ bool CWalletDB::WriteCryptedKey(const CPubKey& vchPubKey,
     return true;
 }
 
+bool CWalletDB::WriteCryptedZKey(const libzcash::PaymentAddress & addr,
+                                 const libzcash::ViewingKey &vk,
+                                 const std::vector<unsigned char>& vchCryptedSecret,
+                                 const CKeyMetadata &keyMeta)
+{
+    const bool fEraseUnencryptedKey = true;
```

Comment:
For parity.  Inherited from upstream method for encrypted t-keys.  In a future release we could remove it since its redundant.